examples/amazon/nginx_server/packages.sh

```diff
@@ -0,0 +1,5 @@
+echo "installing nginx "
+sudo apt-get update
+sudo apt-get install nginx -y
+sudo apt-get update
```

Comment:
why updating after installing ? Here we could also remove the cache from apt-get to free a little space